cmd/dataprep/create.go

```diff
@@ -55,6 +55,12 @@ var CreateCmd = &cli.Command{
 			Value:       "",
 			DefaultText: "Determined by --max-size",
 		},
+		&cli.StringFlag{
+			Name:        "min-piece-size",
+			Usage:       "The minimum size of a piece. Pieces smaller than this will be padded up to this size. It's recommended to leave this as the default",
```

Comment:
Why do we have to pad at all? And are we sure this is compatible with the limits on padding that the Fil+ program set? Whatever that limit is, I think it would be good to either document it here, or check elsewhere to ensure that `min-piece-size` isn't set high enough to cause an issue.